test/navigation/imu_preprocessing.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include <core/logger.hpp>
+#include <core/types.hpp>
+#include <navigation/preprocess_imu.hpp>
+#include <utils/manual_time.hpp>
+
+namespace hyped::test {
+
+core::Float epsilon = 1e-5;
+bool checkArrayEquality(core::ImuData &imu_data_a, core::ImuData &imu_data_b)
+{
+  if (imu_data_a.size() == imu_data_b.size()) {
+    for (std::size_t i; i < imu_data_a.size(); ++i) {
+      if (!(std::abs(imu_data_a.at(i) - imu_data_b.at(i)) < epsilon)) { return false; }
+    }
+  } else {
+    return false;
+  }
+  return true;
+}
+
+void test()
```

Comment:
what is this good for?

Reply:
This was just to instantiate the class to see if it compiled before the other tests were made, will just delete this now as the class is getting instantiated in every other test